### PR TITLE
unity quickstart updates

### PR DESCRIPTION
**Pull request by hussain-khalid**

### CODE REVIEW

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 1]
```diff
@@ -73,68 +73,56 @@ To implement this user interface, in your Unity project:
 
 Import the necessary .NET libraries, set up your <Vpl k="CLIENT" /> to run on Android, and request permissions for the camera and microphone.
 
-1. **Create a new script to hold the system logic**
+1. **Create a new script and import the required Unity libraray**
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:76]: It is a unity library, in that, it works in unity, However, you are importing an Agora SDK

Please change to an import <Vg name="whatever the tag is for video sdk" /> please.

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 2]
```diff
@@ -73,68 +73,56 @@ To implement this user interface, in your Unity project:
...
-    #endif
-    ```
+        ``` csharp
+        #if (UNITY_2018_3_OR_NEWER && UNITY_ANDROID)
+        private ArrayList permissionList = new ArrayList() { Permission.Camera, Permission.Microphone };
+        #endif
+        ```
 
-    3. Check permissions are granted. Add the following code after `Update`:
+    3. Check permissions are granted. In your script file, add the following code after the `Update` method:
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:105]: add the following code after `Update`:

please

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 3]
```diff
@@ -73,68 +73,56 @@ To implement this user interface, in your Unity project:
...
-    ```
+        ```
 
 4. **Bind your script to the canvas**
 
-    1. In `Assets/Agora-RTC-Plugin/Agora-Unity-RTC-SDK/Code`, drag and drop `JoinChannelVideo.cs` to **Canvas** on the **Hierarchy**.
+    1. In `Assets/Agora-RTC-Plugin/Agora-Unity-RTC-SDK/Code`, drag and drop `NewBehaviourScript.cs` to **Canvas** on the **Hierarchy**.
 
-    1. Click **Inspector**, you see that `JoinChannelVideo.cs` is added to **Canvas**.
+    1. Click **Inspector**, you see that `NewBehaviourScript.cs` is added to **Canvas**.
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:125]: that your script is

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 4]
```diff
@@ -308,9 +296,9 @@ To implement this logic, take the following steps:
     ```
     </ProductWrapper>
 
-1. **Join a channel to start <Vpl k="NAME" />**
+1. **Join a channel to start <Vpd k="PRODUCT" />**
 
-    When the local user clicks the `Join` button, you call `Join()`. This method securely connects the local user to a channel using the authentication token. Add the following lines after `InitEventHandler`:
+    When the local user clicks the `Join` button, you call `Join()`. This method securely connects the local user to a channel using the authentication token. In your script file, add the following lines after the `InitEventHandler` method:
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:301]: after `InitEventHandler`:

Global commet. Please don't write the `whatever` method. It is in code, we know it is a method. 

after `methodname`: 

is enough

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 5]
```diff
@@ -471,7 +459,7 @@ In this implementation, you initiate and remove  <Vg k="ENGINE" /> when the app
 
 2. **Start <Vpd k="PRODUCT" />**
 
-    Replace `Start` with the following code:
+    In your script file, replace the `Start` method with the following:
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:462]: replace `Start` with ... 

This is the last time I will comment, please do the others yourself.

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx [hunk 6]
```diff
@@ -483,7 +471,7 @@ In this implementation, you initiate and remove  <Vg k="ENGINE" /> when the app
 
 3. **Clean up the resources used by your <Vpl k="CLIENT" />**
 
-    When a user closes the <Vpl k="CLIENT" />, use `OnApplicationQuit` to clean up the resources you created in `SetupVideoSDKEngine` method. To implement this logic, add the following code at the end of `JoinChannelVideo` class:
+    When a user closes the <Vpl k="CLIENT" />, use `OnApplicationQuit` to clean up the resources you created in `SetupVideoSDKEngine` method. To implement this logic, in your script file, add the following at the end of `NewBehaviourScript` class:
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:474]: funnily enough, leave the class bit :-)

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there.

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx (hunk 1 above)
Comment (hussain-khalid) [reply shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:76]: I did not import Agora SDK. I just explained how to create a script file and which library to import:

1. In **Project**, open **Assets** > **Agora-RTC-Plugin** > **Agora-Unity-RTC-SDK** > **Code**. Right-click **Code**, then click **Create** > **C# Script**. In **Assets** you see the `NewBehaviourScript` file that you use to implement <Vpd k = "PRODUCT"/> in your <Vpl k = "CLIENT"/>.

    1. In **Inspector**, click **Open**. `NewBehaviourScript.cs` opens in your default text editor.

    1. In `NewBehaviourScript.cs`, add the following namespace to the list of namespace declaration:

        ``` csharp
        using UnityEngine.UI;
        ```

File: shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx (hunk 6 above)
Comment (hussain-khalid) [reply shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx:474]: Done

**Pull request merged**